packages/capabilities/package.json

```diff
@@ -23,44 +23,29 @@
     "rc": "npm version prerelease --preid rc"
   },
   "exports": {
-    ".": "./src/index.js",
-    "./*": "./src/*.js",
+    ".": {
+      "types": "./dist/src/index.d.ts",
+      "import": "./src/index.js"
+    },
+    "./*": {
+      "types": "./dist/src/*.d.ts",
+      "import": "./src/*.js"
+    },
     "./types": "./dist/src/types.d.ts"
   },
   "typesVersions": {
     "*": {
-      "provider": [
-        "dist/src/provider"
-      ],
-      "space": [
-        "dist/src/space"
-      ],
-      "store": [
-        "dist/src/store"
-      ],
-      "top": [
-        "dist/src/top"
-      ],
-      "upload": [
-        "dist/src/upload"
-      ],
-      "voucher": [
-        "dist/src/voucher"
-      ],
-      "access": [
-        "dist/src/access"
-      ],
-      "aggregate": [
-        "dist/src/aggregate"
-      ],
-      "offer": [
-        "dist/src/offer"
-      ],
-      "utils": [
-        "dist/src/utils"
+      "*": [
+        "*",
+        "dist/*",
+        "dist/src/*",
+        "dist/src/*/index"
       ],
-      "types": [
-        "dist/src/types"
+      "src/*": [
+        "*",
+        "dist/*",
```

Comment:
Doesn't this cover bottom two ?

Reply:
My understanding is that for a file at `src/foo/bar/baz` this line would ensure tsc looks in `dist/foo/bar/baz` while the lines below would ensure it looked in `dist/src/foo/bar/baz` and `dist/src/foo/bar/baz/index` so I do think they're differentiated. 

That said I think some of these are unnecessary and I'd like to slim them down soon - I cargo culted this from the `w3up-client` `package.json` and want to ship as-is since it's working, but will take another look at this post-merge...